lib/__tests__/fixtures/rule-empty-line-before.html

```diff
@@ -5,6 +5,9 @@
 h1 {
```

Comment:
Would the following?

```css
@media (min-width: 100px) {
  h1 {
    font-weight: normal;
  }
}
```

And no warnings be a more appropriate test for this change as your fixing false positives, right?